index.md

```diff
@@ -4,12 +4,13 @@ permalink: /
 
 {% include base.html %}
 
+# Mission and Vision
+
 <div class="container-medium">
   <div class="grid-item">
-    We are Bitspossessed, a software developers collective. We support people, organizations, and communities to realize technologically-driven ideas.<br><br>
+    We are bitspossessed, a software developers collective dedicated to fostering an inclusive and enjoyable environment where individuals can learn together and collaboratively realize technologically-driven ideas. Our mission is to empower people, organizations, and communities by building innovative solutions while prioritizing creativity, playfulness, and mutual growth.<br><br>
```

Comment:
Same here :) Let's discuss this change in a meeting

Reply:
I like the simplicity of the sentence that was before